WebDriverAgentLib/Utilities/FBConfiguration.m

```
@@ -343,6 +346,44 @@ + (NSString *)dismissAlertButtonSelector
   return FBDismissAlertButtonSelector;
 }
 
+#if !TARGET_OS_TV
++ (void)setScreenshotOrientation:(NSString *)orientation
+{
+  // Only UIInterfaceOrientationUnknown is over iOS 8. Others are over iOS 2.
+  // https://developer.apple.com/documentation/uikit/uiinterfaceorientation/uiinterfaceorientationunknown
+  if ([orientation.lowercaseString isEqualToString:@"portrait"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationPortrait;
+  } else if ([orientation.lowercaseString isEqualToString:@"portraitupsidedown"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationPortraitUpsideDown;
+  } else if ([orientation.lowercaseString isEqualToString:@"landscaperight"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationLandscapeRight;
+  }else if ([orientation.lowercaseString isEqualToString:@"landscapeleft"]) {
+    FBScreenshotOrientation = UIInterfaceOrientationLandscapeLeft;
+  } else {
+    FBScreenshotOrientation = UIInterfaceOrientationUnknown;
```

I'd rather throw an error if an unknown setting value is provided